### DISCUSSION TITLE
iserver breaks CVE-2021-44228.yaml

### DISCUSSION THREAD

**Discussion by Sim4n6** [projectdiscovery/nuclei-templates#3330] [Q&A] [answered]
### Nuclei Version:
latest 

### Template file:
https://github.com/projectdiscovery/nuclei-templates/blob/master/cves/2021/CVE-2021-44228.yaml

### Command to reproduce:

just run nuclei using your personal "iserver".

**Comment by ehsandeep** [comment, marked as answer]
@Sim4n6 thank you for flagging this, I've created a feature request based on this information - https://github.com/projectdiscovery/nuclei/issues/1360, currently, we can not write a dynamic matcher to reference interactsh domain, we will update the existing template as soon as we add support for this, currently if you wish to run template with self-hosted interacatsh server, just update the template locally with your interactsh domain.

**Comment by dogasantos** [comment]
+ 1

**Reply by dogasantos** [reply]
`([a-z0-9\.\-]+)\.([a-z0-9]+)\.([a-z0-9]+)\.\w+$` should also work (ugly/expensive regex... but works)

**Reply by ehsandeep** [reply]
updated here - https://github.com/projectdiscovery/nuclei-templates/releases/tag/v8.7.3 @dogasantos thank you for sharing this, @Sim4n6 FYI.
